code/modules/mob/living/stat_states.dm

```diff
@@ -10,10 +10,12 @@
 	add_attack_logs(src, null, "Fallen unconscious", ATKLOG_ALL)
 	log_game("[key_name(src)] fell unconscious at [atom_loc_line(get_turf(src))]")
 	set_stat(UNCONSCIOUS)
+	ADD_TRAIT(src, TRAIT_FLOORED, STAT_TRAIT)
+	ADD_TRAIT(src, TRAIT_IMMOBILIZED, STAT_TRAIT)
+	ADD_TRAIT(src, TRAIT_HANDS_BLOCKED, STAT_TRAIT)
```

Comment:
Do we need to add `TRAIT_KNOCKEDOUT` in addition to these, because we're falling asleep?

Reply:
all `TRAIT_KNOCKEDOUT` is call this proc. so no